a2a/README.md

```
@@ -0,0 +1,137 @@
+
+| Prefix | Why |
+|---|---|
+| `engine::*` | iii engine internals |
+| `state::*` | KV plumbing |
+| `stream::*` | channel plumbing |
+| `iii.*` | SDK callbacks |
+| `a2a::*` | this worker's own JSON-RPC entry |
+
+### CLI flags
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)